app/configurator/configurator-state.tsx

```diff
@@ -1223,14 +1224,16 @@ export const ConfiguratorStateProvider = ({
 }) => {
   // Ensure that the state is reset by using the `chartId` as `key`
   return (
-    <ConfiguratorStateProviderInternal
-      key={chartId}
-      chartId={chartId}
-      initialState={initialState}
-      allowDefaultRedirect={allowDefaultRedirect}
-    >
-      {children}
-    </ConfiguratorStateProviderInternal>
+    <ChartErrorProvider>
+      <ConfiguratorStateProviderInternal
+        key={chartId}
+        chartId={chartId}
+        initialState={initialState}
+        allowDefaultRedirect={allowDefaultRedirect}
+      >
+        {children}
+      </ConfiguratorStateProviderInternal>
+    </ChartErrorProvider>
```

Comment:
Needed to access `useChartError` from both chart-type-selector and charts themselves. I can also move this to some other part of the app, I am not sure where is the best place to do this :)